### PR TITLE
feat: Implement community RFC-34

### PR DESCRIPTION
**Pull request by Xuanwo**
Signed-off-by: Xuanwo <github@xuanwo.io>

### CODE REVIEW

File: services/github.go [hunk 1]
```diff
@@ -347,29 +505,27 @@ func (g *Github) listTeamMembers(ctx context.Context, team string) (users []stri
...
+		if err == nil {
+			// The team is exist, we can continue to setup next team.
+			continue
+		}
 
-			if resp.StatusCode != 404 {
-				// This error is not a valid github error, return directly.
-				return fmt.Errorf("get team by slug %s: %v", slug, err)
-			}
+		if resp.StatusCode != 404 {
```
Comment (Prnyself) [inline services/github.go:516]: missing error non-nil check here

File: services/github.go [hunk 2]
```diff
@@ -3,6 +3,7 @@ package services
 import (
 	"context"
 	"fmt"
+	"github.com/gobwas/glob"
```
Comment (Prnyself) [inline services/github.go:6]: sort the import plz

**Review by Prnyself — CHANGES_REQUESTED** [review]

File: services/github.go (hunk 1 above)
Comment (Xuanwo) [reply services/github.go:516]: err != nil is expected.

Comment (Prnyself) [reply services/github.go:516]: Emmm, this looks weird.

**Review by Prnyself — APPROVED** [review]

**Pull request merged**